PIL/Image.py

```diff
@@ -632,6 +677,7 @@ def _repr_png_(self):
 
     @property
     def __array_interface__(self):
+        # type: () -> Dict[Text, Any]
```

Comment:
+1 on `any` here for now, this is an implementation of an interface defined elsewhere for datatype->datatype conversion, it's not something that's going to get hit in ordinary development., so type hints for IDEs aren't going to be essential.